readme.md

```diff
@@ -42,6 +42,13 @@ That way you are in control of the order in which things are executed.
 
 
 ## Testing
+There should be unit tests for your code. A unit test should be independent and do an assert for each unit. The test is independent so it has no dependencies on the database or external services. An exception is a repository test -- there you want to test a database transaction.
```

Comment:
"The test is independent so it has no dependencies on the database or external services"
Shouldn't that be: "The tests for business logic should be independent so it has no dependencies on database or external services"